tests/unit/test_advanced.py

```diff
@@ -57,11 +57,16 @@ def initial_feedback(cls):
         return cls.expected_configuration()["initial_feedback"]
 
     def test_get_configuration(self):
-        self.assertEqual(self.expected_configuration(), self.block.get_configuration())
+        try:
+            self.assertEqual(self.expected_configuration(), self.block.get_configuration())
+        except AssertionError:
+            print self.expected_configuration()
+            print self.block.get_configuration()
+            raise
```

Comment:
@itsjeyd both - it is a leftover, but it is a useful leftover, because diffs are very hard to read, so most of the time it is simpler if you have those dicts printed.

CC @mtyaka @bradenmacdonald what do you think about this?